tests/system/test_system.py

```diff
@@ -46,11 +47,13 @@ def _get_credentials_and_project():
     if FIRESTORE_EMULATOR:
         credentials = EMULATOR_CREDS
         project = FIRESTORE_PROJECT
-    else:
+    elif FIRESTORE_CREDS:
         credentials = service_account.Credentials.from_service_account_file(
             FIRESTORE_CREDS
         )
         project = FIRESTORE_PROJECT or credentials.project_id
+    else:
```

Comment:
would like someone from @googleapis/yoshi-python to take a look at this specifically

Reply:
@kolea2 did anyone from @googleapis/yoshi-python review this? I can't tell from the conversation.

Reply:
I believe @nicain did!